agent/php_agent.c

```diff
@@ -654,6 +654,43 @@ zval* nr_php_get_class_constant(const zend_class_entry* ce, const char* name) {
 #endif
 }
 
+char* nr_php_get_object_constant(zval* app, char* name) {
```

Comment:
Since `name` does not change and `nr_php_get_class_constant` expect a `const char *` for it's second argument, suggest making this a `const char *` for consistency.

Reply:
Fixed: [be9c572](https://github.com/newrelic/newrelic-php-agent/pull/775/commits/be9c5728e2763a3c0546ecd9ed8930fb0599d4cf)